staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -181,12 +181,12 @@ func GetPrefixTransformers(config *ResourceConfig) ([]value.PrefixTransformer, e
 	return result, nil
 }
 
-// BlockTransformerFunc takes an AES cipher block and returns a value transformer.
-type BlockTransformerFunc func(cipher.Block) value.Transformer
+// blockTransformerFunc takes an AES cipher block and returns a value transformer.
+type blockTransformerFunc func(cipher.Block) value.Transformer
```

Comment:
I decided to make some functions private, it doesn't relate to my change. Let me know if it's inappropriate change at all or if I should submit it separately.

Reply:
there's no real reason to make these private, so let's revert this change.